recipe/meta.yaml

```diff
@@ -11,70 +11,179 @@ source:
   patches:
     - build.patch
 
+# The "true" build is specified in the _dvc output below.
+# Of the build properties here and below:
+# noarch is not inherited and must be specified per output
+# number is propagated into the outputs below
 build:
-  number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  skip: true  # [py<36]
```

Comment:
Did we lose this?

Reply:
I *think* conda-forge doesn't target `py<36` anymore, so we weren't seeing a delta in the build matrix, but I've reintroduced for the sake of clarity in the recipe.